tools/client_package_initializer.py

```diff
@@ -105,8 +113,23 @@ def _load_setup(self) -> None:
 
         self._setup_data.update(setup)
 
+    @classmethod
+    def _append_missing_requirements(cls, setup: Dict[str, List]):
+        # for assisted_service_api.py
+        if "kubernetes" not in "\n".join(setup[Setup.INSTALL_REQUIRES_KEY]):
+            setup[Setup.INSTALL_REQUIRES_KEY].append("kubernetes~=18.20.0")
+        if "waiting" not in "\n".join(setup[Setup.INSTALL_REQUIRES_KEY]):
+            setup[Setup.INSTALL_REQUIRES_KEY].append("waiting~=1.4.1")
+        if "requests" not in "\n".join(setup[Setup.INSTALL_REQUIRES_KEY]):
+            setup[Setup.INSTALL_REQUIRES_KEY].append("requests~=2.26.0")
+        if "urllib3" not in "\n".join(setup[Setup.INSTALL_REQUIRES_KEY]):
+            setup[Setup.INSTALL_REQUIRES_KEY].append("urllib3~=1.26.7")
+        if "retry" not in "\n".join(setup[Setup.INSTALL_REQUIRES_KEY]):
+            setup[Setup.INSTALL_REQUIRES_KEY].append("retry~=0.9.2")
```

Comment:
there must be a shorter way to write this code

Reply:
Create another requirements file and update the current one